docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,503 @@
+---
+title: "Plant Watering Robot with a Raspberry Pi"
+linkTitle: "Watering Robot"
+weight: 50
+type: "docs"
+description: "Create a plant watering robot with a Raspberry Pi single-board computer."
+tags: ["raspberry pi", "app", "board", "motor"]
+# SMES: Olivia Miller
+---
+
+![Picture of the plant watering robot on a desk.](../../img/plant-watering-pi/plant-watering-robot.png)
+
+Building a useful robot doesn't have to require complicated code or expensive equipment.
+With a Raspberry Pi and some cheap, basic hardware, you can keep your plants healthy and happy from anywhere in the world!
+
+Follow this tutorial to learn how to set up an automatic plant watering system:
+
+1.  [Complete the physical assembly and wiring](/#set-up-your-plant-watering-robot).
+2. [Create and connect to the robot](#configure-the-components-of-your-robot-in-the-viam-app).
+3. [Configure your robot's components](#configure-the-components-of-your-robot-in-the-viam-app).
+4. [Configure a custom Capacitive Soil Moisture sensor component](#configure-the-capacitive-soil-moisture-sensor-as-a-custom-sensor-component-model).
+5. [Write code utilizing the Viam Python SDK to control the plant watering robot](#add-python-control-code).
+
+The tutorial uses the following hardware, but you can adjust it as needed:
+
+- A Raspberry Pi with SD card
+- A [Capacitive Soil Moisture Sensor](https://www.amazon.com/KeeYees-Sensitivity-Moisture-Watering-Manager/dp/B07QXZC8TQ/ref=asc_df_B07QXZC8TQ/?tag=hyprod-20&linkCode=df0&hvadid=343238573411&hvpos=&hvnetw=g&hvrand=14606440922488452520&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9067609&hvtargid=pla-757549749596&psc=1&tag=&ref=&adgrpid=71762478951&hvpone=&hvptwo=&hvadid=343238573411&hvpos=&hvnetw=g&hvrand=14606440922488452520&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9067609&hvtargid=pla-757549749596)
```

Comment:
nit: any amazon URLS can be shortened by removing from ref= and beyond

Reply:
thank you, was not aware of this! Done